src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -24,7 +26,23 @@
 BLUE = 123.0
 
 
-def analyze_video(video_path, time_range=None):
+def analyze_video(video_path, time_range=None) -> Dict:
+    """Analyses video finding faces, given videopath and a timerange
+
+    Parameters
+    ----------
+    video_path
+        the path of the video
+    time_range
```

Comment:
You can add the type that time_range is (if you know it) or at the very least add "optional" as it has a default of None